### PR TITLE
Fix bug caused by a dumb programmer

### PR DESCRIPTION
**Pull request by adamsong**
# Document the changes in your pull request

Geoffrey stickman should not be performing admin actions

# Changelog

:cl:  
bugfix:MFA reset log no longer uses IC name
/:cl:

### CODE REVIEW

**Comment by JamieD1** [comment]
> # Document the changes in your pull request
> Geoffrey stickman should not be performing admin actions
> 
> # Changelog
> 🆑 bugfix:MFA reset log no longer uses IC name /🆑

Why not, hes a centcom colonel

File: code/modules/admin/permissionedit.dm [hunk 1]
```diff
@@ -44,8 +44,8 @@
 				return
 			if(tgui_alert(usr, "If you have been requested to reset the MFA credentials for someone, please confirm that you have verified their identity. Resetting MFA for an unverified person can result in a breach of server security.", "Confirmation", list("I Understand", "Cancel")) != "I Understand")
 				return
-			message_admins("MFA for [admin_ckey] has been reset by [usr]!")
-			log_admin("MFA Reset for [admin_ckey] by [usr]!")
+			message_admins("MFA for [admin_ckey] has been reset by [ADMIN_LOOKUPFLW(usr)]!")
```
Comment (JamieD1) [inline code/modules/admin/permissionedit.dm:47]: ADMINLOOKUPFLW?

That includes follow

Comment (adamsong) [reply code/modules/admin/permissionedit.dm:47]: Yes it does

**Comment by Hopekz** [comment]
You and your PR are toxic.
You're actually insulting the volunteer contributor during a bugfix 👀

The title should represent what your PR does vs using it to take a jab at others.

File: code/modules/admin/permissionedit.dm (hunk 1 above)
Comment (JamieD1) [reply code/modules/admin/permissionedit.dm:47]: Why do I need to follow someone doing an admin task

**Pull request merged**